.github/workflows/build.yml

```diff
@@ -11,41 +11,53 @@ on:
 
 jobs:
   verify-python:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, ubuntu-24.04]
+        scapy_version: [2.5.0, 2.6.1]
+      # Don't abort other runs when one of them fails, to ease debugging.
+      fail-fast: false
+
     name: Python code verification (src)
```

Comment:
I would call this linting or static analysis. Minor comment.

I updated the CI to require these tests now.